tests/composite_pk/tests.py

```diff
@@ -158,6 +158,14 @@ def test_query(self):
         users = User.objects.values_list("pk").order_by("pk")
         self.assertNotIn('AS "pk"', str(users.query))
 
+    def test_raw(self):
+        users = User.objects.raw("SELECT * FROM composite_pk_user")
+        self.assertEqual(len(users), 1)
+        user = users[0]
+        self.assertEqual(user.tenant_id, self.user.tenant_id)
+        self.assertEqual(user.id, self.user.id)
+        self.assertEqual(user.email, self.user.email)
```

Comment:
It might be worth us adding a test to show that having just one of the pks is not enough
```python
    def test_raw_missing_PK_fields(self):
        query = "SELECT tenant_id, email FROM composite_pk_user"
        msg = "Raw query must include the primary key"
        with self.assertRaisesMessage(FieldDoesNotExist, msg):
            list(User.objects.raw(query))
```